src/modules/vendor/components/programs/ProfileContent.vue

```diff
@@ -163,11 +163,17 @@ export default {
     };
   },
   computed: {
-    ...mapState('program', ['program']),
+    ...mapState('program', ['program', 'openedStep']),
   },
   async mounted () {
```

Comment:
created

Reply:
je pense que 
```
    if (!this.program) await this.refreshProgram();
    await this.refreshProgramList();
```
est a mettre dans created
mais pour l'ajout de Kenny avec le `if (this.openedStep)`, il faut le laisser dnas le mounted

Reply:
Si je fais ta solution, le scroll ne fonctionne plus. Si je mets tout dans created ça fonctionne et si je mets tout dans mounted ça fonctionne. J'ai préféré donc ne pas toucher à mounted pour l'instant

Reply:
s'il faut tout laisser d'un meme bloc, c'est plutot a mettre dans le created